datadog_checks_base/datadog_checks/base/utils/tracing.py

```diff
@@ -118,42 +176,46 @@ def tracing_enabled():
 
 
 def traced_class(cls):
-    integration_tracing, integration_tracing_exhaustive = tracing_enabled()
-    if integration_tracing:
```

Comment:
@coignetp while discussing this with the team, I got asked why we're dropping the conditional for this variable.

Reply:
@iliakur dropping this and moving it in `configure_tracer` because we want to be able to enable it at runtime